### ISSUE TITLE
OC is malware?!

### ISSUE THREAD

**Issue by n00b7** [MightyPirates/OpenComputers#1717]
https://www.virustotal.com/en/file/2a1da1cc7dee1727bcb7681d17a4bbf16be3cf9f0ba1f7f5028cec60048fb34d/analysis/

i scanned the latest version of this mod with virustotal and it says 1/56
PE:Malware.Generic(Thunder)!1.A1C4 [F]
what is PE:Malware.Generic(Thunder)!1.A1C4 [F] ???

all versions seem to have this virus

**Comment by LizzyTrickster** [comment]
Which site did you download the mod from?

**Comment by n00b7** [comment]
the opencomputers project on curseforge

**Comment by Kubuxu** [comment]
It probably is a false positive, especially that only one scanner tags it.

Quick read on the internet shows that `Rising` is quite trigger happy with false positives.

**Comment by fnuecke** [comment]
Yeah, that scanner is most certainly being an idiot. Partially evident by the gloriously specific classification of "Malware.Generic". I mean, seriously? -.-

There's no malware of any kind in OC, unless Curse f*cked with the JAR I uploaded, which I doubt very much :P The OC JAR is pretty big, so I suppose chances are it just found some byte sequence it has in its database as "evil" from a different context, which is harmless in this case.

For giggles, [I tried to throw Forge at it](https://www.virustotal.com/en/file/c3f97c1f20a5b40fa4be73671a10414d28234dce785cec38865a64fa2e5bbb22/analysis/), and guess what!? 1/57 think it's a trojan! So yeah, don't buy into snake oil.
